### DISCUSSION TITLE
request support for pinterest

### DISCUSSION THREAD

**Discussion by teel-writ-ros** [delucis/astro-embed#267] [Feature Requests]
request support for the ability to embed pinterest posts like the following.

`<iframe src="https://assets.pinterest.com/ext/embed.html?id=7036943160995828" height="550" width="450" frameborder="0" scrolling="no" ></iframe>`

thanks

**Comment by delucis** [comment]
Thanks for the request. Noting that in general the aim for this library is zero or low-JS components for better web performance, so we probably wouldn’t implement it just as an `<iframe>` or using Pinterest’s default client bundle.

Instead, maybe it’s possible to use the Pinterest API to fetch data about a post (e.g. https://developers.pinterest.com/docs/api/v5/pins-get/) and display that content.

**Comment by teel-writ-ros** [comment]
> Thanks for the request. Noting that in general the aim for this library is zero or low-JS components for better web performance, so we probably wouldn’t implement it just as an `<iframe>` or using Pinterest’s default client bundle.
> 
> Instead, maybe it’s possible to use the Pinterest API to fetch data about a post (e.g. https://developers.pinterest.com/docs/api/v5/pins-get/) and display that content.

certainly, looking forward to an implementation other than <iframe>, which doesn't work for now btw.
